### ISSUE TITLE
Warning when using autoconf from version 2.62 and 2.70

### ISSUE THREAD

**Issue by gsnw** [schweikert/fping#284]
Autoconf generate the following warning when using from version 2.62
Fix is in progress

```
configure.ac:7: warning: The macro `AC_GNU_SOURCE' is obsolete.
configure.ac:7: You should run autoupdate.
./lib/autoconf/specific.m4:312: AC_GNU_SOURCE is expanded from...
configure.ac:7: the top level
configure.ac:62: warning: The macro `AC_PROG_CC_STDC' is obsolete.
configure.ac:62: You should run autoupdate.
./lib/autoconf/c.m4:1666: AC_PROG_CC_STDC is expanded from...
```

**Reference**
[1] https://www.gnu.org/software/autoconf/manual/autoconf-2.62/html_node/Obsolete-Macros.html
[2] https://www.gnu.org/software/autoconf/manual/autoconf-2.70/html_node/Obsolete-Macros.html

**Comment by gsnw** [comment]
For the obsolete macro AC_GNU_SOURCE I have implemented a solution in 65e98a366ceb37c997d76baeb2f1cfad6bbf801c. For AC_PROG_CC_STDC I have not yet found a solution, because this is obsolete only from version 2.70.
Problem with AC_PROG_CC_STDC is, the used AC_AUTOCONF_VERSION is e.g. with Debian too inaccurate, in order to be able to read out a clear version.
This is also described at https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Versioning.html.

**Comment by gsnw** [comment]
Correction is ready with pull request #286
